app/controllers/CheckYourAnswersController.scala

```diff
@@ -70,19 +71,15 @@ class CheckYourAnswersController @Inject()(checkYourAnswersPage: CheckYourAnswer
         reasonableExcuse => {
         if (sessionAnswersHelper.getAllTheContentForCheckYourAnswersPage().nonEmpty) {
           logger.debug(s"[CheckYourAnswersController][onPageLoad] Loading check your answers page")
-          if (reasonableExcuse == "other") {
-            for {
-              previousUploads <- uploadJourneyRepository.getUploadsForJourney(request.session.get(SessionKeys.journeyId))
-            } yield {
-              val previousUploadsFileName = previousUploads.map( _.map( file => file.uploadDetails.map( details => details.fileName)))
-              val answersFromSession = sessionAnswersHelper.getAllTheContentForCheckYourAnswersPage(Some(previousUploadsFileName.getOrElse(Seq.empty).collect{
-                case Some(x) => x
-              }.mkString(", ")))
-              Ok(checkYourAnswersPage(answersFromSession))
-            }
-          } else {
-            val answersFromSession = sessionAnswersHelper.getAllTheContentForCheckYourAnswersPage()
-            Future(Ok(checkYourAnswersPage(answersFromSession)))
+          for {
+            previousUploads <- uploadJourneyRepository.getUploadsForJourney(request.session.get(SessionKeys.journeyId))
+          } yield {
+            val previousUploadsFileName = previousUploads.map(_.map(file => file.uploadDetails.map(details => details.fileName)))
+            val x = previousUploadsFileName.getOrElse(Seq.empty).collect {
```

Comment:
This is my mistake :sorry: . But can you make `x` to be something like `fileNames`?